<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fTelemetryIn</class>
 <widget class="QWidget" name="fTelemetryIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TelemetryIn</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="leDataURL">
         <property name="text">
          <string>225.0.58.1:60058</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbReset">
           <property name="text">
            <string>重置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbConfirm">
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="ckEnabled">
         <property name="text">
          <string>启用</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>命令状态组播地址</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>数据组播地址</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leComStatusURL">
         <property name="text">
          <string>225.0.58.2:60059</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QRadioButton" name="rbLocalControl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>本控</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="rbRemoteControl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>远控</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="rbStartSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>开始存盘</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="rbStopSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>停止存盘</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rbStartSend">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>开始发送</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="rbStopSend">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>停止发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pbStopSave">
         <property name="text">
          <string>停止存盘</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pbStartSend">
         <property name="text">
          <string>开始发送</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pbStartSave">
         <property name="text">
          <string>开始存盘</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pbStopSend">
         <property name="text">
          <string>停止发送</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pbLocalControl">
         <property name="text">
          <string>本控</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pbRemoteControl">
         <property name="text">
          <string>远控</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
